src/App.tsx

```diff
@@ -15,71 +16,108 @@ export const goods = [
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
+      event.currentTarget.classList.add('is-info');
+      event.currentTarget.setAttribute('data-cy', 'RemoveButton');
+      tdActive?.parentElement?.classList.add('has-background-success-light');
+      // eslint-disable-next-line no-param-reassign
+      event.currentTarget.innerText = '-';
+      this.setState({ name: tdActive?.nextSibling?.textContent || '' });
+      this.setState({ selectedItem: true });
+    }
+  };
 
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+  clearAll = () => {
+    const button = document.querySelectorAll('.button');
+    const tr = document.querySelectorAll('tr');
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+    button.forEach((elem) => {
+      elem.classList.remove('is-info');
+      // eslint-disable-next-line no-param-reassign
+      elem.textContent = '+';
+      elem.setAttribute('data-cy', 'AddButton');
+    });
 
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
+    tr.forEach((elem) => elem.classList
+      .remove('has-background-success-light'));
+    this.setState({ selectedItem: false });
+  };
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Garlic
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </main>
-);
+  createMassiveElements = () => {
+    return (
+      goods.map((elem) => {
+        return (
+          <tr
+            data-cy="Good"
+            key={elem}
+            className={
+              classNames({ 'has-background-success-light': elem === 'Jam' })
+            }
+          >
+            <td>
+              <button
+                data-cy={elem === 'Jam' ? 'RemoveButton' : 'AddButton'}
+                type="button"
+                className={classNames('button', { 'is-info': elem === 'Jam' })}
+                onClick={this.controlEvent}
+              >
+                {elem === 'Jam' ? '-' : '+'}
+              </button>
+            </td>
+
+            <td data-cy="GoodTitle" className="is-vcentered">
+              {elem}
+            </td>
+          </tr>
+        );
+      })
+    );
+  };
+
+  render() {
+    return (
+      <main className="section container">
+        {this.state.selectedItem
+          ? (
+            <h1 className="title is-flex is-align-items-center">
+              {`${this.state.name} is selected`}
+
+              {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
```

Comment:
Don't disable eslint

Reply:
Without this "disable" the button requaires attribute lable. This line was write down before my text.